books/free-programming-books-langs.md

```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### Bash
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://drive.google.com/file/d/15LVUS7KI6JB9OmZSrkNiwN1DLHaz8ttS/view) - Carl Albing & JP Vossen (PDF)
```

Comment:
Drive links are not supported. Please provide official website links where this resource can be accessed

Reply:
Btw here's the link https://www.pdfdrive.com/bash-cookbook-solutions-and-examples-for-bash-users-e184776614.html also if you don't wanna merge the book, then please merge only course

Reply:
remove the books